client/src/components/Cart/Item.js

```diff
@@ -0,0 +1,25 @@
+import "../Cart/Item.css";
+
+function Item({ item }) {
+  return (
+    <div id="item-container">
+      <span className="image-container">
+        <img src={item.image} alt="Broken link" />
+      </span>
+      <div>
+        <h1>{item.name}</h1>
+      </div>
+      <span className="price-container">
+        <h1 className="price-value">${item.price}</h1>
+        <br />
+        <br />
```

Comment:
Could we please have a component here using which we can update quantity of a particular item ?